ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # deblur changelog
 
+## Version 1.0.5-dev
```

Comment:
I don't understand why the version number has been incremented, this should still fall under 1.0.4-dev right?